src/Components/Grants/GrantsShow.jsx

```diff
@@ -217,13 +219,18 @@ export default function GrantsShow() {
           breadCrumbLink={`/organizations/${currentOrganization.id}/grants/`}
           copyLink={`/grants/${grant.id}/copy/`}
           editLink={`/grants/${grant.id}/edit/`}
+          setOverView={setOverView}
+          overView={overView}
```

Comment:
this is a nitpick, but maybe we use overview as one word? overview/setOverview?

Reply:
good catch!